File: test/single-language/php/composer/http-service/success-runtime-config/src/SummaryService.php
<?php
declare(strict_types=1);
namespace Fixture;

final class SummaryService
{
    public function summarize(?string $raw): Summary
    {
        $tokens = $raw === null ? ['1', '2', '3'] : explode(',', $raw);
        if (count($tokens) > 20) throw new \InvalidArgumentException('invalid-values');
        $items = [];
        foreach ($tokens as $token) {
            if (!preg_match('/^[0-9]{1,10}$/D', $token)) throw new \InvalidArgumentException('invalid-values');
            $items[] = (int) $token;
        }

        $schema = json_decode('{"type":"array","minItems":1,"maxItems":20,"items":{"type":"integer","minimum":0,"maximum":10000}}', false, 512, JSON_THROW_ON_ERROR);
        if (!(new \Opis\JsonSchema\Validator())->validate($items, $schema)->isValid())
            throw new \InvalidArgumentException('invalid-values');

        return new Summary($items);
    }
}

File: test/single-language/java/jdk/http-service/failure-configuration-rejected/src/acceptance/service/SummaryService.java
package acceptance.service;

import java.util.ArrayList;
import java.util.List;

import acceptance.model.Summary;

public final class SummaryService {
    public Summary summarize(String raw) {
        if (raw == null)
            raw = "1,2,3";
        String[] tokens = raw.split(",", -1);
        if (tokens.length > 20)
            throw new IllegalArgumentException("invalid-values");
        List<Integer> items = new ArrayList<>();
        for (String part : tokens) {
            if (!part.matches("[0-9]{1,10}"))
                throw new IllegalArgumentException("invalid-values");
            long number = Long.parseLong(part);
            if (number > 10000)
                throw new IllegalArgumentException("invalid-values");
            items.add((int) number);
        }
        return new Summary(items);
    }
}
